stage0/src/lib.rs

```diff
@@ -68,6 +68,10 @@ static mut SEV_SECRETS: MaybeUninit<oak_sev_guest::secrets::SecretsPage> = Maybe
 #[no_mangle]
 static SEV_CPUID: MaybeUninit<oak_sev_guest::cpuid::CpuidPage> = MaybeUninit::uninit();
 
+// Will be set in the bootstrap assembly code where we have to read the MSR anyway.
+#[no_mangle]
+static mut SEV_STATUS: SevStatus = SevStatus::empty();
```

Comment:
I assume this has to be mutable so that it ends up in bss rather than rodata, is that correct? Is there perhaps some other way to achieve this without mut to avoid the unsafe block below?

Reply:
It's even worse, if the static is not `mut` I think the optimizer will realize the static is ever supposed to be zero and remove all the code that depends on any of the bits set.

Otherwise I agree; it doesn't need to be `mut`. I guess some wrapper with interior mutability could do the trick, but even then, there'd be some `unsafe` somewhere in there, I think.